overrides/scripts/MysticalAgriculture.zs

```diff
@@ -95,6 +95,108 @@ recipes.addShaped(<bewitchment:juniper_sapling> * 4, [[null,<mysticalagriculture
+
+recipes.addShaped(<divinerpg:eden_sapling> * 4, [[<mysticalagriculture:water_essence>,<mysticalagriculture:water_essence>,<mysticalagriculture:water_essence>],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+recipes.addShaped(<divinerpg:mortum_sapling> * 4, [[<mysticalagriculture:water_essence>,<mysticalagriculture:water_essence>,null],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+recipes.addShaped(<divinerpg:apalachia_sapling> * 4, [[<mysticalagriculture:water_essence>,null,null],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+recipes.addShaped(<divinerpg:wildwood_sapling> * 4, [[<mysticalagriculture:water_essence>,null,<mysticalagriculture:water_essence>],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+recipes.addShaped(<divinerpg:divine_sapling> * 4, [[null,null,<mysticalagriculture:water_essence>],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+recipes.addShaped(<divinerpg:frozen_sapling> * 4, [[null,<mysticalagriculture:water_essence>,<mysticalagriculture:water_essence>],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+recipes.addShaped(<divinerpg:skythern_sapling> * 4, [[<mysticalagriculture:water_essence>,<mysticalagriculture:stone_essence>,<mysticalagriculture:water_essence>],[<mysticalagriculture:nature_essence>,null,<mysticalagriculture:nature_essence>],[null,<mysticalagriculture:wood_essence>,null]]);
+
+# Removing Mystical Agriculture Log Recipes
+for L in [
+    "mysticalagriculture:log",
+    "mysticalagriculture:log2_1",
+    "mysticalagriculture:log_2",
+    "mysticalagriculture:log_1",
+    "mysticalagriculture:log2",
+    "mysticalagriculture:log_3",
+    "mysticalagriculture:menril_log",
+] as string[] {
+    recipes.removeByRecipeName(L);
+}
+
+# Adding Mystical Agriculture Recipes for most Logs
+function addLogRecipes(output as IItemStack, sapling_input as IIngredient){
+    recipes.addShaped(output*16, [[<mysticalagriculture:wood_essence>,<mysticalagriculture:wood_essence>,<mysticalagriculture:wood_essence>],[<mysticalagriculture:wood_essence>,sapling_input,<mysticalagriculture:wood_essence>],[<mysticalagriculture:wood_essence>,<mysticalagriculture:wood_essence>,<mysticalagriculture:wood_essence>]]);
+}
+
+addLogRecipes(<minecraft:log>, <minecraft:sapling>);
```

Comment:
there are a number of saplings (eg from traverse or the twilight forest) which give oak logs. could you add them via `item | item`?

additionally, there are slime saplings from tinkers - perhaps a similar recipe, yet requiring slime essence instead of wood?

Reply:
I think I got them all, let me know if anything else you want